src/components/Autocomplete/OptionsList.jsx

```diff
@@ -0,0 +1,11 @@
+const OptionsList = ({ options }) => {
+  return options.length ? (
+    <ul>
+      {options.map((option) => {
+        return <li>{option.name}</li>;
```

Comment:
Could you add a `key` property for the `<li>`
Not a big deal: in this case we can use arrow function without the `return` statement